<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminLogin</class>
 <widget class="QDialog" name="AdminLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Login</string>
  </property>
  <widget class="QTextEdit" name="usernameText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Username...</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Please Enter a Valid Login Credential:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Log In</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="passViewButton">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="face-wink">
     <activeoff>../../../../Downloads/1564520_code_open_password_icon.png</activeoff>
     <activeon>../../../../Downloads/locked.png</activeon>
    </iconset>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorMessage">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
